src/App.tsx

```diff
@@ -2,108 +1,78 @@
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { getItemsOnPage } from './utils/pagination/getItemsOnPage';
+import { getItemsOnPageCount } from './utils/pagination/getItemsOnPageCount';
+import { Pagination } from './components/Pagination';
+import { EPaginationSize } from './enums/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState<EPaginationSize>(
+    EPaginationSize.Medium,
+  );
+
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const perPageSizeHandler = (e: ChangeEvent<HTMLSelectElement>) => {
+    const pageSize = +e.target.value;
+
+    if (pageSize === perPage) {
+      return;
+    }
+
+    setPerPage(pageSize);
+    setCurrentPage(1);
+  };
+
+  const onPageChange = (page: number) => setCurrentPage(+page);
+
+  const [first, last] = getItemsOnPageCount(currentPage, perPage, items);
```

Comment:
The `getItemsOnPageCount` function is expected to return two numbers representing the first and last item indices on the current page. Ensure that this function is implemented correctly in `./utils/pagination/getItemsOnPageCount` and returns the expected values.